addons/bestja_account_deletion/models.py

```diff
@@ -7,7 +7,7 @@
 class Volunteer(models.Model):
     _inherit = 'res.users'
 
-    active_state = fields.Selection(selection_add=[('deleted', 'usunięty')])
+    status = fields.Selection(selection_add=[('deleted', 'usunięty')])
```

Comment:
`state` not `status`, since `state` is a special field in Odoo, that is supported in other parts of Odoo (for example views).